### ISSUE TITLE
Sublists (<ol> in <ol>)  are not properly rendered

### ISSUE THREAD

**Issue by bvirlet** [helpscout/beacon-ios-sdk#30]
Thank you for taking the time to submit an issue with all the details shown below.  Our engineering team monitors issues submitted and strives to respond with 1-2 business days.

* Beacon install method: [ ] Carthage, [X] CocoaPods, [ ] Manual

**Actual outcome**

Embedded lists are not properly rendered, see the second list starting with "Create contacts".

![image](https://user-images.githubusercontent.com/792706/55140258-e5aecc80-5137-11e9-99c5-e848c0f46f63.png)

**Expected outcome**

The embedded list should be properly rendered: see how "Create contacts" and subsequent items are properly indented indexed with letters.

![Screen Shot 2019-03-28 at 8 58 50 AM](https://user-images.githubusercontent.com/792706/55140281-f95a3300-5137-11e9-99dc-1194045cf457.png)

See the related code:

```
<ol>
	<li><strong>Index</strong> and <strong><a href="https://help.thegrizzlylabs.com/article/240-how-to-search-my-scans-with-their-content">search the content of your documents.</a></strong></li>
	<li>Create <strong>searchable PDFs</strong> – when OCR is enabled, your exported PDFs will include a text layer which means that you will be able to copy their text.</li>
	<li>Extract metadata from your scans and <a href="https://help.thegrizzlylabs.com/article/239-how-to-scan-a-business-card">particularly business cards</a>
	<ol>
		<li><strong>Create contacts</strong> from business cards (or name the document based on the contact name).</li>
		<li>Extract and locate places and addresses.</li>
		<li>Call phone numbers detected in the scan.</li>
		<li>Rename a document based on a date detected in the scan.</li>
	</ol>
	</li>
</ol>
```

**Comment by jgritman** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue with nested <ol> tags for articles](https://trello.com/c/ILoCkYuZ/373-issue-with-nested-ol-tags-for-articles)

**Comment by jgritman** [comment]
Thanks for the bug report Bruno! We'll have a fix for this in our next release.

**Comment by samsymons** [comment]
Hey @bvirlet – this issue should now be fixed in Beacon SDK 1.0.1. Give that a try and let me know if any other issues come up!
